### DISCUSSION TITLE
.net core 3.1 and mvc

### DISCUSSION THREAD

**Discussion by alexkrush** [quartznet/quartznet#1055] [Q&A] [answered]
Hi all,
I'm trying to create a new job in my controller. Could you please share an example for it?
What do I need to put in Startup.Configure function and how can I get a reference to Scheduler in controller:
public IActionResult Index()
        {
......
Many thanks in advance and
rgds
Alexander

**Comment by lahma** [comment, marked as answer]
Check the examples in the source repository, there's one for ASP.NET too. Get a reference to IScheduler and work your magic.
